### DISCUSSION TITLE
no subfolders or files in filesystem (localdisk) and none in remote filesystem

### DISCUSSION THREAD

**Discussion by sonewtogithub** [music-assistant/support#1413] [Q&A] [answered]
I haven audio files on my Synology Nasfolder  "Muziek/muziek" These files are visible in home assistant under /Media/nasfiles

I don't succeed in adding these files in Music Assistant with filesystem(localdisk) . As path I set /media. But when I go to folder Filesystem (local disk) I see subfolder "nasfiles"but no subfolders or files under  /nasfiles.

I shared the folder /Muziek on my Nas and tried to reach it wiht the Filesystem remote share but that does not work either.
I entered following settings:
Server: internal IP-adress of my nasserver
Share : a random name (or has this to be the directory name of the shared folder ? Did not work neither
Username:  I tried with several users, such as a separate user and an admin-account
Password: 
Subfolder: tried several (sub)foldernames but nothing worked.

I am running homeassitant on a virtual machine on Synology 218+
I tried with several users even with an admin-account

**Comment by OzGav** [comment, marked as answer]
The share name can’t be random it has to be the actual name you have setup for the shared folder. When you say “it doesn’t work” please be more specific with what happens. Do you see error messages appear in a dialog box? Are there errors in the log?

**Reply by sonewtogithub** [reply]
Sorry for this late response, have been away for a few days. Amongst several names I used the exact name of the folder too. So now you acknowledged that I had to use that, I had to solve the error "permission denied". Strange error because I have used the correct  credentials.
I know use the credentials of a user with admin-rights and that seems to works Not happy with that. Now I have to find why the user with only read rights does not work, might be a Synology issue..

For now, thank you for pushing me in the right direction.
